client/vue.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  "transpileDependencies": [
+    "vuetify",
+    "vuex-module-decorators"
+  ]
+}
```

Comment:
Note that you can specify Webpack config in here, or set the public path, eg
`publicPath: process.env.VUE_APP_PATH,`
(URL path)